Fix test for django 3.2 migration test_scheduler_carga

Fix test for django 3.2 migration

Related to https://github.com/avdata99/escrutinio-social/pull/1
A PR to the branch release-dj3-py38 is expected

Fix test `elecciones/tests/test_scheduler_carga.py::test_siguiente_prioriza_estado_y_luego_coeficiente - Failed: Expected to perform 17 queries but 14 were done`

El test prueba la cantidad de queries que se hace dentro de un bloque.
Es muy probable que Django haya cambiado

<pre>
Failed: Expected to perform 17 queries but 14 were done
E               
E               Queries:
E               ========
E               
E               SELECT "cache_key", "value", "expires" FROM "elecciones_cache" WHERE "cache_key" IN (':1:PRIORIDAD_STATUS')
E               
E               SELECT "cache_key", "value", "expires" FROM "elecciones_cache" WHERE "cache_key" IN (':1:MULTIPLICADOR_CANT_ASIGNACIONES_REALIZADAS')
E               
E               SELECT "cache_key", "value", "expires" FROM "elecciones_cache" WHERE "cache_key" IN (':1:MULTIPLICADOR_CANT_ASIGNACIONES_REALIZADAS')
E               
E               SELECT "constance_config"."id", "constance_config"."key", "constance_config"."value" FROM "constance_config" WHERE "constance_config"."key" = 'MULTIPLICADOR_CANT_ASIGNACIONES_REALIZADAS' LIMIT 21
E               
E               SELECT "constance_config"."id", "constance_config"."key", "constance_config"."value" FROM "constance_config" WHERE "constance_config"."key" = 'MULTIPLICADOR_CANT_ASIGNACIONES_REALIZADAS' LIMIT 21
E               
E               SAVEPOINT "s132732325544064_x50"
E               
E               INSERT INTO "constance_config" ("key", "value") VALUES ('MULTIPLICADOR_CANT_ASIGNACIONES_REALIZADAS', 'gAJLAi4=') RETURNING "constance_config"."id"
E               
E               RELEASE SAVEPOINT "s132732325544064_x50"
E               
E               SELECT COUNT(*) FROM "elecciones_cache"
E               
E               SAVEPOINT "s132732325544064_x51"
E               
E               SELECT "cache_key", "expires" FROM "elecciones_cache" WHERE "cache_key" = ':1:MULTIPLICADOR_CANT_ASIGNACIONES_REALIZADAS'
E               
E               INSERT INTO "elecciones_cache" ("cache_key", "value", "expires") VALUES (':1:MULTIPLICADOR_CANT_ASIGNACIONES_REALIZADAS', 'gAVLAi4=', '2025-09-11T20:35:23'::timestamp)
E               
E               RELEASE SAVEPOINT "s132732325544064_x51"
E               
E               SELECT "elecciones_mesacategoria"."id", "elecciones_mesacategoria"."status", "elecciones_mesacategoria"."mesa_id", "elecciones_mesacategoria"."categoria_id", "elecciones_mesacategoria"."carga_testigo_id", "elecciones_mesacategoria"."carga_oficial_id", "elecciones_mesacategoria"."parcial_oficial_id", "elecciones_mesacategoria"."percentil", "elecciones_mesacategoria"."orden_de_llegada", "elecciones_mesacategoria"."coeficiente_para_orden_de_carga", "elecciones_mesacategoria"."cant_fiscales_asignados", "elecciones_mesacategoria"."cant_asignaciones_realizadas", CASE WHEN "elecciones_mesacategoria"."status" = 'parcial_en_conflicto' THEN 0 WHEN "elecciones_mesacategoria"."status" = 'parcial_sin_consolidar' THEN 1 WHEN "elecciones_mesacategoria"."status" = 'sin_cargar' THEN 2 WHEN "elecciones_mesacategoria"."status" = 'parcial_consolidada_csv' THEN 3 WHEN "elecciones_mesacategoria"."status" = 'parcial_consolidada_dc' THEN 4 WHEN "elecciones_mesacategoria"."status" = 'total_sin_consolidar' THEN 5 WHEN "elecciones_mesacategoria"."status" = 'total_en_conflicto' THEN 6 WHEN "elecciones_mesacategoria"."status" = 'total_consolidada_csv' THEN 7 WHEN "elecciones_mesacategoria"."status" = 'total_consolidada_dc' THEN 8 WHEN "elecciones_mesacategoria"."status" = 'con_problemas' THEN 9 ELSE '0' END AS "prioridad_status", ("elecciones_mesacategoria"."cant_fiscales_asignados" / 2) AS "cant_fiscales_asignados_redondeados", ("elecciones_mesacategoria"."cant_asignaciones_realizadas" / 4) AS "cant_asignaciones_realizadas_redondeadas" FROM "elecciones_mesacategoria" INNER JOIN "elecciones_categoria" ON ("elecciones_mesacategoria"."categoria_id" = "elecciones_categoria"."id") WHERE (NOT (NOT "elecciones_categoria"."activa") AND "elecciones_mesacategoria"."coeficiente_para_orden_de_carga" IS NOT NULL AND NOT (EXISTS(SELECT (1) AS "a" FROM "elecciones_mesa" U1 LEFT OUTER JOIN "adjuntos_attachment" U2 ON (U1."id" = U2."mesa_id") WHERE (U2."id" IS NULL AND U1."id" = "elecciones_mesacategoria"."mesa_id") LIMIT 1)) AND NOT ("elecciones_mesacategoria"."status" = 'con_problemas') AND NOT ("elecciones_mesacategoria"."status" = 'total_consolidada_dc')) ORDER BY "cant_fiscales_asignados_redondeados" ASC, "prioridad_status" ASC, "elecciones_mesacategoria"."coeficiente_para_orden_de_carga" ASC, "cant_asignaciones_realizadas_redondeadas" ASC, "elecciones_mesacategoria"."id" ASC LIMIT 1 FOR UPDATE SKIP LOCKED
</pre>

--- a/django_app/elecciones/tests/test_scheduler_carga.py
+++ b/django_app/elecciones/tests/test_scheduler_carga.py
@@ -1,21 +1,16 @@
 import pytest
-from datetime import timedelta
-from django.utils import timezone
 
 from elecciones.tests.factories import (
     AttachmentFactory,
     CategoriaFactory,
-    CircuitoFactory,
     FiscalFactory,
     IdentificacionFactory,
-    MesaCategoriaDefaultFactory,
     MesaCategoriaFactory,
     MesaFactory,
 )
-from elecciones.models import MesaCategoria, Mesa
+from elecciones.models import MesaCategoria
 
-from adjuntos.consolidacion import consumir_novedades_identificacion, liberar_mesacategorias_y_attachments
-from problemas.models import Problema, ReporteDeProblema
+from adjuntos.consolidacion import liberar_mesacategorias_y_attachments
 
 
 @pytest.fixture
@@ -75,7 +70,9 @@ def test_siguiente_prioriza_estado_y_luego_coeficiente(db, settings, setup_const
         coeficiente_para_orden_de_carga=2.0,
         mesa=m3
     )
-    with django_assert_num_queries(17):
+    # Este numero depende de caches habilitados y cosas internas de Django
+    # Si hay cambios grandes, actualizarlo
+    with django_assert_num_queries(14):
         assert MesaCategoria.objects.siguiente() == mc1
 
     for i in range(settings.MIN_COINCIDENCIAS_CARGAS):

